### ISSUE TITLE
Docker wrapper with mac M1

### ISSUE THREAD

**Issue by jakepalmer** [PennLINC/qsiprep#297]
Hi,

TLDR: I'm looking for some advice on running qsiprep with docker on a Macbook Pro with M1 chip. I can see on dockerhub that the current docker image is built for linux/amd64 architecture so understand this isn't supported currently. I've also seen a similar [issue](https://github.com/nipreps/fmriprep/issues/2431) with fmriprep and guess the advice may be similar:
>1. Run fMRIPrep on x86_64 chips.
>2. Set up a native installation on your M1.
>3. Work on building an ARM-based Docker image...

I've tried a few things anyway but can't see any options that would avoid introducing breaking changes:
- Running per standard instructions, which results in what looks like a python multiprocessing error (output attached).
-  I looked into the nvidia/cuda base images for cuda version 9.1 and there aren't any. I think the earliest version for arm64 is 11.* with ubuntu 18.04, which would I'm assuming would introduce problems (e.g. with eddy).
- I tried to rebuild the container with the following:
`docker buildx build --load --platform linux/arm64/v8 --tag qsiprep-arm64:test .`
per [these](https://www.docker.com/blog/multi-arch-build-and-images-the-simple-way/) instructions. As expected, this didn't work because `nvidia/cuda:9.1-runtime-ubuntu16.04` base image doesn't support arm64.
- Also tried the above after changing the base image to `nvidia/cuda:11.2.2-runtime-ubuntu18.04` out of interest, though this also didn't build.

Appreciate any advice!
Thanks.
[log.txt](https://github.com/PennLINC/qsiprep/files/7193615/log.txt)

**Comment by jakepalmer** [comment]
Haven't found a solution for this other than using different workstation or building new arm docker image. Closing as just a limitation of hardware for now.
